Microsoft.Toolkit.Uwp.UI/Extensions/Visual/VisualEx.cs

```diff
@@ -593,39 +593,49 @@ private static void OnSizeChanged(DependencyObject d, DependencyPropertyChangedE
             }
         }
 
-        private static void OnKeepCenterPointCenteredChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        private static void OnNormalizedCenterPointChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
         {
             if (d is FrameworkElement element && !DesignTimeHelpers.IsRunningInLegacyDesignerMode)
             {
-                SetupKeepCenterPointCentered(e, element);
+                SetupNormalizedCenterPoint(e, element);
             }
         }
 
-        private static void SetupKeepCenterPointCentered(DependencyPropertyChangedEventArgs e, FrameworkElement element)
+        private static void SetupNormalizedCenterPoint(DependencyPropertyChangedEventArgs e, FrameworkElement element)
         {
             element.SizeChanged -= KeepCenteredElementSizeChanged;
 
-            if (e.NewValue is bool keepCentered)
+            if (e.NewValue is string normalizedValue)
             {
-                if (keepCentered)
+                element.SizeChanged -= KeepCenteredElementSizeChanged;
+                try
                 {
-                    element.SizeChanged -= KeepCenteredElementSizeChanged;
-                    element.SizeChanged += KeepCenteredElementSizeChanged;
+                    var vectorValue = normalizedValue.ToVector3();
+                    if (vectorValue.X < 0 || vectorValue.X > 1 || vectorValue.Y < 0 || vectorValue.Y > 1)
```

Comment:
I would say it would be fine *not* to enforce this - quite a few times I've found it useful put centre points outside of 0 to 1 - for example, setting the centre point of a page header to match that of the actual entire page, or an element on the circumference of a circle having a centre point at the centre of the circle.

Reply:
Good point, removed